core/src/main/scala/kafka/coordinator/transaction/TransactionCoordinator.scala

```diff
@@ -280,7 +280,7 @@ class TransactionCoordinator(brokerId: Int,
   private def logInvalidStateTransitionAndReturnError(transactionalId: String,
                                                       transactionState: TransactionState,
                                                       transactionResult: TransactionResult) = {
-    debug(s"TransactionalId: $transactionalId's state is $transactionState, but received transaction " +
+    info(s"TransactionalId: $transactionalId's state is $transactionState, but received transaction " +
```

Comment:
Since we are adding the info level log after the `match` block I think it is redundant to make this info also.